grawlix-cms/_admin/book.view.php

```diff
@@ -215,7 +215,7 @@
 	$pages_per_view = 50;
 }
 else {
-	$pages_per_view = 70;
+	$pages_per_view = 100; //Why 70?
```

Comment:
I suspect 70 was chosen because 100 makes for a very long page, but 100 makes it easier to actually navigate the pages, so I think this change is fine.

The comment should be removed before we merge this :]